src/api/models/lines.js

```diff
@@ -4,6 +4,8 @@ import idvalidator from 'mongoose-id-validator'
 const Schema = mongoose.Schema;
 const LineEvent = mongoose.model('LineEvent')
 
+const BATCH_SIZE = 5;
```

Comment:
soo i think theres already a batch size in updateAllPrelineToNotification cron job, can u modify that file to reference this BATCH_SIZE?